gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.2-all.zip
```

Comment:
should we get the bin version here as well?

Reply:
bin is the one without source IIRC. `all` comes with everything bundled

Reply:
ignore, I meant the sources.